shared-test-resources/TypeResolverTestSuite/ByteCodeHierarchyTest/src/Class3.java

```diff
@@ -0,0 +1,18 @@
+public class Class3 extends Class2{
+
+    @Override
+    public void method1(){
+        System.out.println("I'm method 1 in Class3");
+    }
+
+    @Override
+    public void method2(){
+        System.out.println("I'm method2 in Class3");
+    }
+
+    @Override
+    public void method3(){
+        System.out.println("I'm method3 in Class3");
```

Comment:
I think adding println to the test case will slow the test down. is this needed?

Reply:
it does not really as the runtime jar is not added to analysisinputlocations :)